src/components/board/Board.vue

```diff
@@ -124,13 +126,51 @@ export default {
 		},
 	},
 	watch: {
-		id: 'fetchData',
+		id(newValue, oldValue) {
+			if (oldValue) {
```

Comment:
This may be skipped if notify push is not enabled

Reply:
with https://github.com/nextcloud/deck/commit/8f698e8c015a7c98898a514c7f0fd31203079d7d the session create&close methods are still being called, but basically noop's when notify push is not enabled